tests/test_nc_to_mmd.py

```diff
@@ -676,5 +676,12 @@ def test_file_on_faked_thredds(self):
         nc2mmd.to_mmd(netcdf_local_path=fn)
         self.assertEqual(nc2mmd.metadata['storage_information']['file_name'], 'reference_nc.nc')
 
+    def test_access_constraint(self):
+        tested = tempfile.mkstemp()[1]
+        nc2mmd = Nc_to_mmd('tests/data/reference_nc.nc', check_only=True)
+        nc2mmd.to_mmd()
+        spatial_repr = nc2mmd.metadata['access_constraint']
+        self.assertEqual(spatial_repr, 'test-access-constraint')
```

Comment:
This is a controlled vocabulary but it is not implemented as that in the xsd. We should anyway use one of the allowed values, like "Open".

Reply:
Solved in a912ad6